Enhancement/NewGlossaryPage

fjtria opened the pull request:
Updated glossary page UI. Went with scroll view on desktop instead of pages. Also replaced individual letter page jumps with ranges instead.

Alex-dev7 reviewed (changes requested):
On mobile or smaller screens can you make the glossary page scroll down more  so that the search bar is almost at the top before entering the inner scroll container with the paragraphs. This way there will be more room for the text on the screen (at the moment it is only half of the screen height).

Suggestion: search bar on desktop view seems too wide, a max width should fix it. Move the dropdown closer to the search bar.

Optional: style the scroll bar from default style to something to match the website theme.(can use the same style as the dropdown for the search bar)

fjtria commented:
- Adjusted the page to scroll until SearchBar was at the top
- Made SearchBar width responsive
- Removed the padding above search results container

The SearchBar scroll styles didn't work for me (found out that webkit scrollbar is non-standard). Instead I added custom a scrollbar style using scrollbar-color (more on track to be standard, still limited support though) to globalStyles to keep it consistent on every page. Colors can be adjusted if needed.

Alex-dev7 reviewed (approved):
Looks good, ready to merge!
